[[ Bug ]] Fix module lcext note to describe correct docs update

montegoulding opened the pull request:
The release note for the changes to static linked code libraries for iOS was
not updated when the documentation changes were moved to the `Extending LiveCode`
guide.

runrevmark commented on docs/notes/feature-modulelcext.md:
```diff
@@ -2,6 +2,6 @@
-Foreign Code Libraries has been added to the LiveCode Builder Language Reference
-describing the creation of `.lcext` objects and use of code libraries in general.
\ No newline at end of file
+`--forcebuiltins --outputauxc` options. Additionaly, the `Using compiled libraries`
```
`Additionaly` should be `Additionally` (two l's)

montegoulding committed:
[[ Bug ]] Fix module lcext note to describe correct docs update

The release note for the changes to static linked code libraries for iOS was
not updated when the documentation changes were moved to the `Extending LiveCode`
guide.

montegoulding commented:
@runrevmark Additionally fixed!
